### DISCUSSION TITLE
Cannot import css-tag from "lit"

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [lit/lit#2555] [Q&A]
"lit": "^2.1.4",
![image](https://user-images.githubusercontent.com/77786952/154417217-1366ae7a-a4f5-4d7e-a851-32cfd1d3c286.png)

I can not import css-tag function from "lit".

How to set styles?

**Comment by justinfagnani** [comment]
The css tag is re-exported from reactive element. I don't know what could be going wrong in your setup.

**Reply by Linya-IronMan** [reply]
I delete all packages and reset "dependencies" in package.sjon, then reinstall lit. Its version change to ^2.2.0， now , It's Okay.
